lib/language_syntax/golang_syntax.dart

```diff
@@ -0,0 +1,57 @@
+// Обработка правильности отступов, точек с запятой для цикла for
+// с учетом комментариев, с учетом строк.
```

Comment:
Please, translate comments that are not in English

Reply:
done